server/v2/helpers/user_helper.js

```diff
@@ -1,17 +1,21 @@
 import bcrypt from 'bcrypt';
 import jwt from 'jsonwebtoken';
 import config from '../config/config';
+import dotenv from 'dotenv';
+dotenv.config();
 const hashPassword = password => bcrypt.hashSync(password, 10);
 const comparePasswords = (plain, encrypted) => {
   return bcrypt.compareSync(plain, encrypted) ? true : false;
 }
-const authenticateUser = ({id,email,first_name}) => {
+const authenticateUser = ({id,email,first_name, is_admin}) => {
+  const { SECRET } = process.env;
   return jwt.sign({
     id,
     email,
-    first_name
+    first_name,
```

Comment:
@WinnersProx You only need the email in the object, you should remove the id, first_name and, last_name and retrieve the user's information when validation the token